.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   system:
     docker:
-      - image: circleci/node:14-browsers
+      - image: circleci/node:16-browsers
```

Comment:
Ah, apparently this image has been replaced: https://circleci.com/developer/images/image/cimg/node

Maybe it's: `cimg/node:16-browsers` now?